mobius-rx/src/main/java/com/spotify/mobius/rx/RxMobius.java

```diff
@@ -57,7 +60,27 @@ private RxMobius() {
    */
   public static <M, E, F> Observable.Transformer<E, M> loopFrom(
       final MobiusLoop.Factory<M, E, F> loopFactory, final M startModel) {
-    return new RxMobiusLoop<>(loopFactory, startModel);
+    return new RxMobiusLoop<>(loopFactory, startModel, Collections.emptySet());
+  }
+
+  /**
+   * Create an observable transformer that starts from a given {@link Init} function.
+   *
+   * <p>Every time the resulting observable is subscribed to, a new MobiusLoop will be started from
+   * the given init function.
+   *
+   * @param loopFactory gets invoked for each subscription, to create a new MobiusLoop instance
+   * @param startModel the starting point for each new loop
+   * @param init the init function to start from each new loop
+   * @param <M> the model type
+   * @param <E> the event type
+   * @param <F> the effect type
+   * @return a transformer from event to model that you can connect to your UI
+   */
+  public static <M, E, F> Observable.Transformer<E, M> loopFrom(
```

Comment:
There should be a test validating that the result of the supplied init function is in fact used when starting the loop.

Reply:
Done, added tests